### PR TITLE
Supports Pathname instances in LOAD_PATH

### PR DESCRIPTION
**Pull request by byroot**
Fixes: https://github.com/byroot/bootscale/issues/1

@grosser for review please.

### CODE REVIEW

File: lib/bootscale/cache_builder.rb [hunk 1]
```diff
@@ -7,7 +7,10 @@ def initialize
     end
 
     def generate(load_path)
-      ordered_entries = load_path.map { |path| entries.fetch(path) { entries[path] = Entry.new(path) } }
+      ordered_entries = load_path.map do |path|
+        path = path.to_s
+        entries.fetch(path) { entries[path] = Entry.new(path) }
```
Comment (grosser) [inline lib/bootscale/cache_builder.rb:12]: `entries[path] ||= Entry.new(path)` is simpler and faster ?

**Comment by grosser** [comment]
:+1: ... but I'll try to clean up our pathnames

File: lib/bootscale/cache_builder.rb (hunk 1 above)
Comment (byroot) [inline lib/bootscale/cache_builder.rb:12]: That's a good point.

Comment (byroot) [inline lib/bootscale/cache_builder.rb:12]: https://github.com/byroot/bootscale/commit/e2a204dc43c54a191e78c5639b61a2dce80d9563

**Pull request merged**